### DISCUSSION TITLE
Issue starting Miningcore

### DISCUSSION THREAD

**Discussion by MiningCryptoLive** [coinfoundry/miningcore#1191] [Q&A]
I am rebuilding with the latest release and get the following error when starting the pool.

Configuration file error: Value "integrated" is not defined in enum. Path 'banning.manager', 

any help would be appreciated

**Comment by Onslivion** [comment]
Value "integrated" needs to be changed to "Integrated"(capital I) in your config file;
```
   "banning": {
        "manager": "Integrated",
         [...]
    }

```
